content/docs/command-reference/metrics/index.md

```diff
@@ -1,47 +1,104 @@
 # metrics
 
-A set of commands to add, manage, collect, and display project metrics:
-[add](/doc/command-reference/metrics/add),
-[show](/doc/command-reference/metrics/show),
-[diff](/doc/command-reference/metrics/diff),
-[modify](/doc/command-reference/metrics/modify), and
-[remove](/doc/command-reference/metrics/remove).
+A set of commands to manage, collect, and display _project metrics_:
+[show](/doc/command-reference/metrics/show), and
+[diff](/doc/command-reference/metrics/diff).
 
 ## Synopsis
 
 ```usage
 usage: dvc metrics [-h] [-q | -v]
-                   {show,add,modify,remove,diff} ...
```

Comment:
remove them from the highlighter

Reply:
Already gone 🙂

Reply:
It's config\prismjs\dvc-commands.js right?

Reply:
yep, name sounds right